Pipfile.lock

```diff
@@ -106,13 +108,46 @@
             ],
             "version": "==18.1.0"
         },
+        "colorama": {
```

Comment:
adding, removing, adding...

Reply:
🤷‍♂️ 💯

Reply:
can you recreate the lock by using the README commands that rely on the `pipenv` in the container to see if it becomes magically consistent?